test/common/testinfra.py

```diff
@@ -661,11 +653,20 @@ def issue_requests_image_refresh(issue, comments, image):
             comments = self.get(issue['comments_url'])
             for image in DEFAULT_IMAGE_REFRESH:
                 if issue_requests_image_refresh(issue, comments, image):
+                    requested[image] = image
```

Comment:
Would be a bit more straightforward to use `requested = set()` and `requested.add(image)`.